unime/package.json

```diff
@@ -11,8 +11,9 @@
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
     "test": "vitest",
     "coverage": "vitest run --coverage",
-    "lint": "prettier . --check && eslint .",
+    "lint": "eslint .",
     "format": "prettier . --write --plugin prettier-plugin-svelte --plugin prettier-plugin-tailwindcss --plugin @ianvs/prettier-plugin-sort-imports",
+    "format:check": "prettier . --check --plugin prettier-plugin-svelte --plugin prettier-plugin-tailwindcss --plugin @ianvs/prettier-plugin-sort-imports",
```

Comment:
Nice catch! 🥳 I had to remove all plugin declarations to make it work, otherwise when you specify at least one explicitly it will overwrite the plugins defined in `.prettierrc`.